CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `values` prop to the `Slider`.
```

Comment:
Nice! Could you add an example to Slider's `README.md`?

Reply:
Done!
What do you think? Is it clear enough?
https://github.com/vtex/styleguide/pull/1365/files#diff-e0ddd676cd7ba2d0660c0c4b6072a93018d44068ffa59471830abb5d4bed872eR35

Reply:
It's not working on [Styleguide's preview](https://styleguide-git-feature-control-slider-values-programmatically.styleguide-core.now.sh/#/Components/Forms/Slider) :/
I **think** it's because it must be only JS.
Also, could you add inputs and state to show it programmatically changing?

![image](https://user-images.githubusercontent.com/15948386/108396200-5a3ff800-71f5-11eb-95ee-245e935cdb02.png)

Reply:
Oh i see. I tough it was just markdown, but we are using styleguidist here

Done! Now it is fancy 💅